play/PlayerRules/Basic Rules.txt

```diff
@@ -581,13 +604,14 @@ to roll either for (15 - FORT) turns, or until they pass.
 Conditions:
 Bleeding - d10/2+1 (drop the remainder) stackable damage per turn. Broken with 
 Medicine Skill check. DC varies. If you have a bleed condition, seek medical aid
-immediately, You CAN Bleed Out, you have been warned.
+immediately, You can Bleed Out.

```

Comment:
I removed it because it seemed a tad unprofessional and it's restating the obvious.